components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -157,10 +157,12 @@
 <config_GM_kappa ocn_forcing="datm_forced_restoring" ocn_grid="EC30to60E2r2">600.0</config_GM_kappa>
 <config_GM_closure>'visbeck'</config_GM_closure>
 <config_GM_baroclinic_mode>3.0</config_GM_baroclinic_mode>
-<config_GM_visbeck_alpha>0.13</config_GM_visbeck_alpha>
-<config_GM_visbeck_max_depth>1000.0</config_GM_visbeck_max_depth>
-<config_GM_visbeck_min_kappa>300.0</config_GM_visbeck_min_kappa>
-<config_GM_visbeck_max_kappa>1200.0</config_GM_visbeck_max_kappa>
+<config_GM_Visbeck_alpha>0.005</config_GM_Visbeck_alpha>
```

Comment:
can this be changed back to 0.13?  This is the value we've been using in coupled runs

Reply:
@vanroekel - yes, I'll take care fo that right now

Reply:
OK, should be good now